src/App.js

```diff
@@ -1,11 +1,22 @@
 import './App.css';
 import React from "react"
-import {LandingPage} from "./pages"
+import {LandingPage,PageNotFound,Login} from "./pages"
+import { Route, Routes } from 'react-router-dom';
+import Mockman from "mockman-js";
 
 function App() {
+
   return (
     <div className="App">
-      <LandingPage/>
+      <Routes>
+        <Route path="/" element={<LandingPage/>}></Route>
+        <Route path="/shopNow" element={<>shop Now</>}></Route>
+        <Route path="/login" element={<Login/>}></Route>
+        <Route path="/wishlist" element={<>Wishlist page</>}></Route>
+        <Route path="/cart" element={<>Cart page</>}></Route>
+        <Route path="/mockman" element={<Mockman/>}></Route>
+        <Route path="*" element={<PageNotFound/>}></Route>
```

Comment:
Does * mean if there is any other path other than the listed paths then, show PageNotFound component?

Reply:
Yes

Reply:
Nice. I did not know this!